pycqed/measurement/waveform_control/circuit_builder.py

```diff
@@ -155,39 +157,117 @@ def get_cz_operation_name(self, qb1=None, qb2=None, op_code=None, **kw):
         else:
             raise KeyError(f'CZ gate "{cz_pulse_name} {qb1} {qb2}" not found.')
 
-    def get_pulse(self, op, parse_z_gate=False):
+    def get_pulse(self, op, parse_rotation_gates=False):
         """
         Gets a pulse from the operation dictionary, and possibly parses
         logical indexing as well as arbitrary angle from Z gate operation.
         Examples:
+             >>> get_pulse('CZ 0 2', parse_z_gate=True)
+             will perform a CZ gate (according to cz_pulse_name)
+             between the qubits with logical indices 0 and 2
              >>> get_pulse('Z100 qb1', parse_z_gate=True)
              will perform a 100 degree Z rotation
+             >>> get_pulse('Z:theta qb1', parse_z_gate=True)
+             will perform a parametric Z rotation with parameter name theta
+             >>> get_pulse('Z:2*[theta] qb1', parse_z_gate=True)
+             will perform a parametric Z rotation with twice the
+             value of the parameter named theta. The brackets are used to
+             indicated the parameter name. This feature has also been tested
+             with some more complicated mathematical expression.
+        Adding 's' (for simultaneous) in front of an op_code (e.g.,
```

Comment:
For clarity, I would also add that spaces have a logical meaning, as I could otherwise see people adding spaces in the mathematical expression. Something like:
"Note: the mathematical expression should be entered without any spaces in between operands. For instance, 'Z:2*[theta]/180+[theta]**2 qb1' and not 'Z: 2 * [theta] / 180 + [theta]**2 qb1'"